### PR TITLE
[Documentation] iOS Simulator version&udid

### PR DESCRIPTION
**Pull request by furkanarabaci**
Changes:

1. Change double quote to single quote in order to escape paranthesis
2. Add iOS version command
3. Add udid command

### CODE REVIEW

**Review by cipolleschi — APPROVED** [review]

File: docs/running-on-simulator-ios.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ Once you have your React Native project initialized, you can run `npx react-nati
...
-You can specify the device the simulator should run with the `--simulator` flag, followed by the device name as a string. The default is `"iPhone 13"`. If you wish to run your app on an iPhone SE (2nd generation), run `npx react-native run-ios --simulator="iPhone SE (2nd generation)"`.
+You can specify the device the simulator should run with the `--simulator` flag, followed by the device name as a string. The default is `"iPhone 13"`. If you wish to run your app on an iPhone SE (2nd generation), run `npx react-native run-ios --simulator='iPhone SE (2nd generation)'`.
 
 The device names correspond to the list of devices available in Xcode. You can check your available devices by running `xcrun simctl list devices` from the console.
+
+### Specifying a version of device
+
+If you have multiple iOS versions installed, you also need to specify it's appropiate version. E.g. run `npx react-native run-ios --simulator='iPhone 13 Pro (15.5)'` in order to specify the iOS version.
+
+## Specifying an udid
```
Comment (Simek) [inline docs/running-on-simulator-ios.md:20]:
```suggestion
## Specifying an UDID
```

Correct case.

File: docs/running-on-simulator-ios.md [hunk 2]
```diff
@@ -9,6 +9,14 @@ Once you have your React Native project initialized, you can run `npx react-nati
...
+
+You can specify the device udid returned from `xcrun simctl list devices` command. E.g. run `npx react-native run-ios --udid='AAAAAAAA-AAAA-AAAA-AAAA-AAAAAAAAAAAA'`
```
Comment (Simek) [inline docs/running-on-simulator-ios.md:22]:
```suggestion
You can specify the device UDID returned from `xcrun simctl list devices` command. E.g. run `npx react-native run-ios --udid='AAAAAAAA-AAAA-AAAA-AAAA-AAAAAAAAAAAA'`.
```

Case  + period.

**Review by Simek — COMMENTED** [review]
I have lest few small corrections, but besides that it LGTM. 👍 

Pleas also sign in the CLA, otherwise we cannot merge your PR.

**Comment by furkanarabaci** [comment]
@Simek I was on vacation, sorry about that. Signed the CLA, it says it should be passing a few minutes later. Also thank you for correcting the case, I wasn't sure if I should use lowercase or uppercase.

**Review by Simek — APPROVED** [review]
LGTM, thank you for the update! 👍

**Pull request merged**